.github/workflows/main.yml

```diff
@@ -283,3 +283,23 @@ jobs:
             screens/**
             /tmp/nginx*.log
         if: always()
+
+  build-dev:
+    runs-on: ubuntu-latest
+
+    needs: build
+
+    if: ${{ github.event.pull_request.merged }}
+
+    steps:
+
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Repository Dispatch to the Development image
```

Comment:
This step as it's formulated now, always applies. I think we're currently only dependent on `cpanfile` for correct population of the dev-docker image. In that case, I'd propose we only rebuild when changes get committed to `cpanfile`. What do you say?